# Service to get raw forces in hand
---
int64[] forces      # palm forces
int64 ret           # Set to 1 (success) or 0 (error)
string msg          # Error description
